### PR TITLE
Update for aioredis-1.0

### PR DESCRIPTION
**Pull request by akhilman**
Call create_redis_pool instead of create_pool
All tests passed, example works fine.

Fixes upstream bug #8

### CODE REVIEW

**Comment by akhilman** [comment]
Actually I can make patch that will work with both versions.

**Comment by joanvila** [comment]
Hey @akhilman this is great! It was in my todo but I've been busy recently. Can you update the version to 0.2.0 and the changelog?
About the patch, it would be super nice to have it working with both versions. If that's not possible we should document in a clear way from which version is airoedis 1.0 supported and inform that the new 0.2.0 version requires airoredis 1.0.

**Comment by akhilman** [comment]
Done

**Comment by akhilman** [comment]
Should I move `crate_redis_pool` to `Instance` class or leave it as module function?

File: aioredlock/redis.py [hunk 1]
```diff
@@ -14,14 +16,29 @@ def __init__(self, host='localhost', port=6379, db=0, password=None):
         self._pool = None
         self._lock = asyncio.Lock()
 
+    @staticmethod
+    async def _create_redis_pool(*args, **kwargs):
+        """
+        Adaptor to support both aioredis-0.3.0 and aioredis-1.0.0
```
Comment (joanvila) [inline aioredlock/redis.py]: Typo: Adapter

**Comment by joanvila** [comment]
I'd leave the `crate_redis_pool` function as a static method inside `Instance` as you did. No hard feelings on that

**Comment by joanvila** [comment]
Once you fix the typo I'll merge, push to pypi and review the rest of PRs

**Comment by akhilman** [comment]
Ups, I screwed up all my PR branches and can not update PR.

**Comment by akhilman** [comment]
Let me close this PR and make now one with typo fix.

**Comment by akhilman** [comment]
Please take a look #27

**Pull request closed without merging**